api/graphql/resources/resource_filtersets.py

```diff
@@ -0,0 +1,26 @@
+import django_filters
+
+from permissions.helpers import can_manage_resources, get_units_with_permission
+from spaces.models import Space
+
+
+class ResourceFilterSet(django_filters.FilterSet):
+    only_with_permission = django_filters.BooleanFilter(
+        method="get_only_with_permission"
+    )
+
+    def get_only_with_permission(self, qs, property, value):
+        """Returns resources where the user has any kind of permissions"""
```

Comment:
Text is a bit off since in below there are called `get_units_with_permission(user, "can_manage_resources")` => so I'd say this could read; `Returns resources where the user has resource management permissions.`

Reply:
Fixed! 👍